### ISSUE TITLE
add a fix or better warning when an unquoted date range field is passed to create datatable

### ISSUE THREAD

**Issue by srearl** [CAPLTER/capeml#29]

**Comment by srearl** [comment]
posssible solution...
```r
my_fun <- function(x) {

  y <- substitute(x)
  y <- as.character(y)
  print(y)

}

my_fun("a")
my_fun(a)
```

**Comment by srearl** [comment]
fixed with release 2.0 15d26e63b2867a02a9b738aed9f8934212fb1955
